manifest.php

```diff
@@ -27,7 +27,7 @@
     'label' => 'Tao Offline',
     'description' => 'An extension to assist the setup of an offline context. Setup synchronisation and encryption by test center identifier',
     'license' => 'GPL-2.0',
-    'version' => '2.4.2',
+    'version' => '2.4.2.1',
```

Comment:
as we have `$this->skip('2.3.0', '3.1.0');` in the updater , and we don't have release of `2.4.3`, I would suggest to use 2.4.3 version for bugfix instead of hack that is not described in the semver

Reply:
I've changed the version for backport to `2.4.3` as you've proposed, but I don't think it's a good convention, it isn't a regular extension release in my opinion.